.github/workflows/build.yml

```diff
@@ -301,15 +301,15 @@ jobs:
       # TODO: Once test/Npgsql.Specification.Tests work, switch to just testing on the solution
       - name: Test
         run: |
-            dotnet test -c ${{ matrix.config }} -f ${{ matrix.test_tfm }} test/Npgsql.Tests --logger "GitHubActions;report-warnings=false" --blame-hang-timeout 30s
-            dotnet test -c ${{ matrix.config }} -f ${{ matrix.test_tfm }} test/Npgsql.DependencyInjection.Tests --logger "GitHubActions;report-warnings=false"
+            dotnet test -c ${{ matrix.config }} -f ${{ matrix.test_tfm }} --no-build --no-progress --project test/Npgsql.Tests --hangdump --hangdump-timeout 30s
```

Comment:
Note 1: GitHubActions logger of VSTest doesn't yet support MTP. Not sure if you consider the lack of it as a blocker. Alternatively, you can report TRX files and use https://github.com/marketplace/actions/publish-test-results instead.